crates/next-api/src/operation.rs

```diff
@@ -1,18 +1,24 @@
 use anyhow::Result;
```

Comment:
This file wasn't being used before... I accidentally merged it as part of #70242, but we need this now, so it's finally being used.